MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+recursive-include opentracing *
+recursive-include example *.py
+recursive-include example *.thrift
+recursive-include tests *.py
```

Comment:
manifest is for pypy upload, right?  Perhaps tests/examples are not needed there.